src/.env.example

```
@@ -16,4 +16,8 @@ VITE_ENABLE_DEBUG_LOGS=false
 # Feature Flags
 VITE_ENABLE_ANALYTICS=true
 VITE_ENABLE_ALERTS=true
-VITE_ENABLE_LIVE_STREAM=true
\ No newline at end of file
+VITE_ENABLE_LIVE_STREAM=true
+
+# Theme Settings
+# VITE_DEFAULT_THEME=system  # Options: light, dark, system
+# VITE_THEME_STORAGE_KEY=logpulse-theme
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)